packages/prism-theme-odyssey/src/utils/CustomPropertyInspector.js

```diff
@@ -104,35 +105,35 @@ class CustomPropertyInspector {
           <p>The Custom Property Inspector has nothing to show. This is probably because you are <a href="https://github.com/w3c/csswg-drafts/issues/1316">using Google Chrome.</a></p>
           <p>Try this page on Safari or Firefox!</p>
         </div>
-      `
-      inspector.appendChild(errorNotice)
+      `;
+      inspector.appendChild(errorNotice);
     }
 
     // Prepend the inspector
     document.body.prepend(inspector);
 
     inspectorReset.addEventListener('click', () => {
       Storage.removeItem(LOCALSTORAGE_ID);
-      this.setCustomProperties(args.propertiesElement);
-    })
+      this.setCustomProperties(this.args.propertiesElement);
+    });
 
-    // TODO: Consider listening only input element events 
-    // which are explicitly concerned with the value of the
+    // TODO: Consider listening only input element events
+    // which are explicitly concerned with the value of the
```

Comment:
Interesting.. I can't tell what the diff is here with the space. 🤔

Reply:
it was an [irregular whitespace violation](https://eslint.org/docs/rules/no-irregular-whitespace)